scripts/ansible/install-ansible.sh

```diff
@@ -10,6 +10,10 @@ DIR=$(dirname "$0")
 if which yum > /dev/null; then
     yum install git python3-pip -y
 elif which apt-get > /dev/null; then
+    echo "Checking for dpkg locks..."
+    while fuser /var/lib/dpkg/lock > /dev/null 2>&1 || fuser /var/lib/dpkg/lock-frontend > /dev/null 2>&1; do
```

Comment:
Is there a chance this loop gets stuck without another exit condition/max retries?

Reply:
Given that it's a while loop, absolutely. But that would be due to a host issue where a process maintaining use of those lock files and the subsequent apt update command would fail.

On Jenkins we have timeouts for our builds so that would not result in a stuck build. But for users running this manually it is possible to get stuck.

If we want to do something about this situation, I could add a timeout and fail the entire script if the while loop never exits. What do you think of that?

Reply:
Implemented a 10 minute timeout now.

Reply:
Works for me, thanks!